code/src/cljs/sixsq/nuvla/ui/edges/views.cljs

```diff
@@ -885,7 +1020,6 @@
 (defn NuvlaBoxesOrClusters
   []
   (dispatch [::events/init])
-  ;; (dispatch [::events/refresh-root])
```

Comment:
commented?

Reply:
I removed the commented dispatch.

What do you mean exactly?